### PR TITLE
PREAPPS-6193 : Display signed/encrypted message properly.

### PR DESCRIPTION
**Pull request by BhavinBathani**
Receiver will be able to read signed/encrypted message.
Server does the decryption and provide decrypted content in case of server-smime.
Display info that the message was Signed/Encrypted while viewing email.
Display info if server responded with exception.

### CODE REVIEW

File: src/normalize/entities.ts [hunk 1]
```diff
@@ -198,6 +198,44 @@ MimePart.addMapping({
...
+});
+
+const SmimeCertsIssuedTo = new Entity({
+	c: 'country',
+	cn: 'commonName',
+	emailAddress: 'emailAddress',
+	o: 'organizationName',
+	ou: 'organizationUnit',
+	st: 'state'
+});
```
Comment (lavegupta) [inline src/normalize/entities.ts:225]: `
const commonSmimeCertsFields = {
	c: 'country',
	cn: 'commonName',
	emailAddress: 'emailAddress',
	o: 'organizationName',
	st: 'state'
};

const SmimeCertsIssuedBy = new Entity({
    ...commonSmimeCertsFields,
	l: 'locality'
});

const SmimeCertsIssuedTo = new Entity({
    ...commonSmimeCertsFields,
	ou: 'organizationUnit'
});`

File: src/normalize/entities.ts [hunk 2]
```diff
@@ -198,6 +198,44 @@ MimePart.addMapping({
...
+	emailAddress: 'emailAddress',
+	l: 'locality',
+	o: 'organizationName',
+	st: 'state'
+});
+
+const SmimeCertsIssuedTo = new Entity({
+	c: 'country',
+	cn: 'commonName',
+	emailAddress: 'emailAddress',
```
Comment (silentsakky) [inline src/normalize/entities.ts:221]: no need to specify if key and value are same

**Review by silentsakky — APPROVED** [review]

File: src/normalize/entities.ts (hunk 2 above)
Comment (BhavinBathani) [reply src/normalize/entities.ts]: Fixed.

File: src/normalize/entities.ts (hunk 1 above)
Comment (BhavinBathani) [reply src/normalize/entities.ts]: Fixed.

**Review by silentsakky — APPROVED** [review]

**Review by k-kato — APPROVED** [review]

**Review by lavegupta — APPROVED** [review]

**Review by silentsakky — APPROVED** [review]

**Pull request merged**